lib/index.js

```diff
@@ -14,6 +14,8 @@
       extend = require('./utils/extend'),
       isDateString = require('./utils/assert-date-string');
 
+  handleWindowResize();
```

Comment:
When would this function get called when you load keen-dataviz? Is there a better place to put this function so it's more clear?